apps/web/components/v2/apps/CalendarListContainer.tsx

```diff
@@ -193,10 +193,11 @@ function ConnectedCalendarsList(props: Props) {
                 ) : (
                   <Alert
                     severity="warning"
-                    title={t("calendar_error")}
-                    message={item.error?.message}
+                    title={t("something_went_wrong")}
+                    message={t("calendar_error")}
+                    iconClassName="h-10 w-10 ml-2 mr-1 mt-0.5"
```

Comment:
Made the icon the same size and position as the rest of the icons in the calendar integration list